db/seeds.rb

```diff
@@ -47,12 +47,12 @@
 
 if Region.all.empty?
   puts 'Loading regions'
-  Rake::Task['shf:load_regions'].invoke
+  Rake::Task['shf:load_regions'].execute
```

Comment:
Thanks for the explanation Ashley. I knew about the dependencies, but because the tasks still worked after changing #invoke to #execute I assumed they didn't have any. I see now that they rely on :environment. Have to look into why that even worked (?) . But I'll revert to #invoke and use another solution to solve the problem.

My point about #invoke not working in testing is about something that had me stumped for a while. Because Rake is build as a Make tool, it doesn't invoke a task if it already has been invoked earlier. Thats makes sense for a Make tool: there is no reason to compile something twice. During testing however, we seed a few times. But the shf:load_regions and shf:load_kommuns will only be executed once.

You can reproduce it easily if you change the #execute back to #invoke, and run my tests. All but the first one will fail. However, executing the other test on their own will make them pass.